sidebars.js

```diff
@@ -777,11 +777,6 @@ const sidebars = {
               id: 'how-to-run-applications',
               label: 'ScalarDL Ledger を介して ScalarDL アプリケーションを実行',
             },
-            {
-              type: 'doc',
-              id: 'getting-started-auditor',
-              label: 'ScalarDL Ledger と Auditor を通じてコントラクトを実行',
```

Comment:
Just a confirmation. Is it OK to just remove? Should we update it to "how-to-run-applications-with-auditor"?

Reply:
@jnmt Good catch, thank you! I forgot to add `how-to-run-applications-with-auditor` to the Japanese sidebar navigation after we merged the PR in the internal docs repository. I've added it in 781868ebc19d7cd6e7e7efde1dfef4e3f66b87a7. PTAL!